package.json

```diff
@@ -38,8 +38,10 @@
     "configstore": "^5.0.0",
     "editor": "^1.0.0",
     "enquirer": "^2.3.0",
+    "form-data": "^2.3.3",
     "gray-matter": "^4.0.1",
     "isemail": "^3.1.3",
+    "node-fetch": "^2.6.1",
```

Comment:
So excited to dive into this! Wanna do the honors of uninstalling `request` and `request-promise-native`?